deploy/traction/values-pr.yaml

```diff
@@ -59,6 +59,45 @@ ui:
     active: true
     authority: https://dev.loginproxy.gov.bc.ca/auth/realms/digitaltrust-nrm
     jwksUri: https://dev.loginproxy.gov.bc.ca/auth/realms/digitaltrust-nrm/protocol/openid-connect/certs
+    reservationForm: |
```

Comment:
Is this to be JSON in the YAML file or should it be YAML? Wondering Ivan's opinion.

Reply:
Yes... this is supposed to be JSON

Reply:
👍